plugin/mutate/test/test_admin.d

```diff
@@ -168,4 +168,28 @@ unittest {
     ]).shouldBeIn(r.stderr);
 }
 
-// TODO: add test for plain-report for marked mutants.
+@(testId ~ "shall notify lost marked mutant")
+unittest {
+    // arrange
+    mixin(EnvSetup(globalTestdir));
+    makeDextoolAnalyze(testEnv).addInputArg(testData ~ "fibonacci.cpp").run;
+    MutationId id = 3.to!MutationId;
+    Status toStatus = Status.killedByCompiler;
+    string rationale = `"Lost"`;
+
+    // act
+    makeDextoolAdmin(testEnv)
+        .addArg(["--operation", "markMutant"])
+        .addArg(["--id",        to!string(id)])
+        .addArg(["--to-status", to!string(toStatus)])
+        .addArg(["--rationale", rationale])
+        .run;
+    auto r = makeDextoolAnalyze(testEnv).addInputArg(testData ~ "abs.cpp").run;
+
+    // assert
+    testAnyOrder!SubStr([
+        "| ID | File                                              | Line | Column | Status           | Rationale |",
+        "|----|---------------------------------------------------|------|--------|------------------|-----------|",
+        "| 3  | build/plugin/mutate/plugin_testdata/fibonacci.cpp | 8    | 10     | killedByCompiler | \"Lost\"    |",
```

Comment:
I did at first, but the output in the test, i.e what is captured in r.stdout, had the escaped quotes. Just remade the check so that it worked accordingly.